docs/openfaas-pro/sso/cli.md

```diff
@@ -68,6 +68,9 @@ Next, set the following environment variable `export BROWSER=wslview` in your `~
 
 Then, whenever a browser is launched, it will open on your Windows host, and the result will come back to the WSL environment.
 
+!!! note
+        Services running on WSL can not be accessed using the loopback address `127.0.0.1` on the host. The `auth` command will use `http://localhost:31111/oauth/callback` as the callback when running on WSL. Make sure to register this url as a valid callback url with your OIDC provider.
```

Comment:
Please re-write this to be less confusing.

Use a note header "Note for Windows Subsystem for Linux (WSL) users"

When authenticating from WSL, the address will be changed from X to Y. Make sure that the address: `http://localhost:31111/oauth/callback` is added as a callback URL for the IdP app for OpenFaaS.